packages/discord-request/package.json

```diff
@@ -27,6 +27,7 @@
 	"dependencies": {
 		"discord-error": "0.0.2",
 		"discord-snowflake": "2.0.0",
-		"debug": "^4.3.4"
+		"debug": "^4.3.4",
+		"redis": "^4.3.1"
```

Comment:
Should this be a peer dependency to avoid downloading a package the user will never use?

Reply:
probably! no clue how to set that up though :p